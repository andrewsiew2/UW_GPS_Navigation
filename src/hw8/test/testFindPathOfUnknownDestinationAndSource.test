# Verify that the program handles the case if the destination and source are unknown buildings
r
CSEA
OOG

# All of your test scripts should end with q to terminate the program.
q