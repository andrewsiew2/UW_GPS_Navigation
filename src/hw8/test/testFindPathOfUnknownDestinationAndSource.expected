Menu:
	r to find a route
	b to see a list of all buildings
	q to quit

Enter an option ('m' to see the menu): # Verify that the program handles the case if the destination and source are unknown buildings
Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: CSEA
Unknown building: OOG

Enter an option ('m' to see the menu): 
# All of your test scripts should end with q to terminate the program.
